sal_interfaces/ATCamera/ATCamera_Telemetry.xml

```diff
@@ -239,6 +239,17 @@
         <Units>milliampere</Units>
         <Count>1</Count>
     </item>
+    <!-- Temporary AuxTel only addition for CAP-423. A better long-term solution
+        would be to have an "HVBias on" configuration setting, plus code
+        which will enter fault state if the switch found to be off when
+        it should be on -->
+     <item>
+        <EFDB_Name>hvbiasSwitch</EFDB_Name>
+        <Description>1=On, 0=Off</Description>
+        <IDL_Type>float</IDL_Type>
```

Comment:
Make this a long. Float isn't necessary.

Reply:
All of our CCS telemetry is floats (actually doubles), including this element. It would take special code to convert this one element to a long, and since I consider ti just a temporary workaround it is not worth the effort.

Reply:
Fair point. Looks fine otherwise.